React-Projects/job-portal/package.json

```diff
@@ -10,12 +10,15 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^2.2.3",
```

Comment:
@codal-kcolaco please check what is different between old redux and new redux-toolkit

Reply:
@abhi-codal  Yes